.golangci.yml

```diff
@@ -1,79 +1,124 @@
-       - '(*github.com/valyala/bytebufferpool.ByteBuffer).WriteString' # always returns nil error
+    check-type-assertions: true
+    check-blank: true
+    disable-default-exclusions: true
+    exclude-functions:
+      - '(*bytes.Buffer).Write' # always returns nil error
+      - '(*github.com/valyala/bytebufferpool.ByteBuffer).Write' # always returns nil error
+      - '(*github.com/valyala/bytebufferpool.ByteBuffer).WriteByte' # always returns nil error
+      - '(*github.com/valyala/bytebufferpool.ByteBuffer).WriteString' # always returns nil error
 
   errchkjson:
     report-no-exported: true
 
   exhaustive:
+    check-generated: true
     default-signifies-exhaustive: true
 
   forbidigo:
     forbid:
-      - ^(fmt\.Print(|f|ln)|print|println)$
-      - 'http\.Default(Client|Transport)'
+      - ^print(ln)?$
+      - ^fmt\.Print(f|ln)?$
+      - ^http\.Default(Client|ServeMux|Transport)$
       # TODO: Eventually enable these patterns
-      # - 'time\.Sleep'
-      # - 'panic'
+      # - ^panic$
+      # - ^time\.Sleep$
+    analyze-types: true
+
+  gci:
+    sections:
+      - standard
+      - prefix(github.com/gofiber/fiber)
+      - default
+      - blank
+      - dot
+      # - alias
+    custom-order: true
+
+  goconst:
+    numbers: true
 
   gocritic:
+    # TODO: Uncomment the following lines
+    # enabled-tags:
+    #   - diagnostic
+    #   - style
+    #   - performance
+    #   - experimental
+    #   - opinionated
     disabled-checks:
-      - ifElseChain
+      - ifElseChain # TODO: Do not disable
+      - hugeParam
+      - rangeExprCopy
+      - rangeValCopy
+    settings:
+      captLocal:
+        paramsOnly: false
+      elseif:
+        skipBalanced: false
+      underef:
+        skipRecvDeref: false
+      # NOTE: Set this option to false if other projects rely on this project's code
+      unnamedResult:
+        checkExported: false
 
   gofumpt:
     module-path: github.com/gofiber/fiber
     extra-rules: true
 
   gosec:
     excludes:
-      - G104 # Provided by errcheck
+      - G104 # TODO: Enable this again. Mostly provided by errcheck
     config:
       global:
+        # show-ignored: true # TODO: Enable this
         audit: true
-  
-  depguard:
-    rules:
-      main:
-        deny:
-          - pkg: flag
-            desc: '`flag` package is only allowed in main.go'
-          - pkg: io/ioutil
-            desc: '`io/ioutil` package is deprecated, use the `io` and `os` package instead'
 
   govet:
-    check-shadowing: true
     enable-all: true
     disable:
-      - shadow
       - fieldalignment
-      - loopclosure
+      - shadow
 
   grouper:
+    # const-require-grouping: true # TODO: Enable this
     import-require-single-import: true
     import-require-grouping: true
+    # var-require-grouping: true # TODO: Conflicts with gofumpt
+
+  loggercheck:
+    require-string-key: true
+    no-printf-like: true
```

Comment:
is this relevant for us?

Reply:
I think so, see their examples at https://github.com/timonwong/loggercheck/commit/ef578c4cc83f7962d4521b1db8f67e75136c25f5#diff-dbd79b07e7f11adb8139d961bff8a7df5f2e3f331e92e9d13d8f34ff84595cdeR1